rebar.config

```diff
@@ -43,7 +43,9 @@
     {stdout_formatter, "0.2.3"},
     %% use specific cuttlefish commit until either 2.2.1 or 2.3.0 is relased.
     {cuttlefish, {git, "https://github.com/Kyorai/cuttlefish.git", {tag, "v2.3.0"}}},
-    {vernemq_dev, {git, "https://github.com/gojekfarm/vernemq_dev.git", {branch, "master"}}},
+    {vernemq_dev,
+        {git, "https://github.com/VivekPipaliya23/vernemq_dev.git",
```

Comment:
CHange it back to stable.

Reply:
[chore: add stable gojek-farm version](https://github.com/gojek/vernemq/pull/51/commits/26f0928bea8fd51f06ff640ccc169c2b5ed0ac12)